pyproject.toml

```diff
@@ -52,10 +52,11 @@ tqdm = "^4.66.6"
 transformers = "^4.46.0"
 torchmetrics = "^1.5.1"
 safetensors = "^0.4.4"
-onnx = "1.16.1"
+onnx = "^1.17.0"
```

Comment:
Please use version 1.16.1 for the moment (https://github.com/sacdallago/biotrainer/issues/111)

Reply:
Ok, sure. The reason I updated the version of onnx as well as added werkzeug was that when I pushed the changes and the GitHub actions triggered, there was an error in the tests, mentioning that I should use onnx ^1.17.0 and werkzeug ^3.0.6. Unfortunately, I deleted the actions workflow of this specific error. But when I made these changes, the problem was solved. However, it seems I was the only one experiencing these problems, so I'll change them back to the original versions to see if they work (which I think they should).